pkgs/development/python-modules/orange3/default.nix

```diff
@@ -92,6 +92,7 @@ let
       pandas
       pyqtwebengine
       serverfiles
+      setuptools
```

Comment:
Either add the package to `propagatedBuildInputs` or `nativeBuildInputs`. Don't keep it in both.

Reply:
OK